test/Libraries/DynamoPythonTests/PythonEvalTestsWithLibraries.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Numerics;
+using Dynamo;
+using Dynamo.Configuration;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.Scheduler;
+using NUnit.Framework;
+using static DSIronPythonTests.IronPythonTests;
+
+namespace DynamoPythonTests
+{
+    public class PythonEvalTestsWithLibraries : DynamoModelTestBase
+    {
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("ProtoGeometry.dll");
```

Comment:
why do we need protogeo for this test?

Reply:
We don't actually. I just figured this may come in handy for the tests we would add.